docs/how_to_release.md

```diff
@@ -0,0 +1,34 @@
+# OverlayPlugin Release Process
+
+This doc describes how to make a release of OverlayPlugin.
+
+## Overview
+
+Anybody can (and should) propose a release by sending a PR if they think a release is needed.
+Only [maintainers](https://github.com/orgs/OverlayPlugin/teams/maintainers/members) can currently publish a release.
```

Comment:
Might want to check the URL and/or the permissions of the maintainer list (if that's even a thing that can be set). I am getting a 404 error when accessing https://github.com/orgs/OverlayPlugin/teams/maintainers/members
...or maybe the team just hasn't been setup yet?

Reply:
Out of curiosity, does it show anything now? I put myself as a public member, whereas before everyone was private

Reply:
Still 404

Reply:
Yeah, I don't see any visibility settings.  It says the team is public.  It doesn't let me look at it without being logged in, but I don't have another account to use that's not me to see what that looks like.  I'll remove the link.